desktop/extensions-sdk/dev/overview.md

```diff
@@ -1,62 +0,0 @@
----
-title: Overview
```

Comment:
I see this page was removed, but there's no replacement or redirect; was that intentional?

Reply:
Yes, the content from this page was distributed to various other pages

Reply:
I added a redirect to the https://docs.docker.com/desktop/extensions-sdk/ landing-page; users should likely be able to find where to go from there if they happened to have that old URL 👍